### ISSUE TITLE
Error `Cannot find module 'ts-dedent'`

### ISSUE THREAD

**Issue by mkosir** [storybookjs/eslint-plugin-storybook#103]
**Describe the bug**
When trying to include eslint-plugin-storybook into .eslint configuration
```
  "extends": [
    ...
    "plugin:storybook/recommended"
  ],
```
 and running eslint, following error is returned:
`Error: Failed to load plugin 'storybook' declared in '.eslintrc': Cannot find module 'ts-dedent'`

**Comment by yannbf** [comment]
Hey @mkosir thank you so much for catching this, I'm fixing it right away!

**Comment by yannbf** [comment]
@mkosir the fix was released in `0.6.3`. Could you try it out? Thanks!

**Comment by mkosir** [comment]
@yannbf that was fast ⚡, thx the new release fixes it!
